heat/core/manipulations.py

```diff
@@ -1392,6 +1399,67 @@ def pad(array, pad_width, mode="constant", constant_values=0):
     return padded_tensor
 
 
+def ravel(a):
+    """
+    Return a flattened view of `a` if possible. A copy is returned otherwise.
+
+    Parameters
+    ----------
+    a : DNDarray
+        array to collapse
+
+    Returns
+    -------
+    ret : DNDarray
+        flattened array with the same dtype as a, but with shape (a.size,).
+
+    Notes
+    ------
+    Returning a view of distributed data is only possible when `split != 0`. The returned DNDarray may be unbalanced.
+    Otherwise, data must be communicated among processes, and `ravel` falls back to `flatten`.
+
+
+    See Also
+    --------
+    :function:`~heat.core.manipulations.flatten`
+
+    Examples
+    --------
+    >>> a = ht.ones((2,3), split=0)
+    >>> b = ht.ravel(a)
+    >>> a[0,0] = 4
+    >>> b
+    DNDarray([4., 1., 1., 1., 1., 1.], dtype=ht.float32, device=cpu:0, split=0)
+    """
+    sanitation.sanitize_in(a)
+
+    if a.split is None:
+        return factories.array(
+            torch.flatten(a._DNDarray__array),
+            dtype=a.dtype,
+            copy=False,
+            is_split=None,
+            device=a.device,
+            comm=a.comm,
+        )
+
+    # Redistribution necessary
+    # Arrays are not perfectly distributed. Array are copied between processes.
+    if a.split != 0:
+        return flatten(a)
```

Comment:
Would an exception be better here?

Reply:
i dont think we need an exception, what is the case for putting one in?

Reply:
To differentiate it from flatten. On second thought, it's good as it is.